dev/releases/2.1.18.md

```diff
@@ -48,7 +48,7 @@ TiDB Ansible 版本：2.1.18
     - 修复 COM_STMT_FETCH 慢日志时间记录和 MySQL 不一致问题 [#12953](https://github.com/pingcap/tidb/pull/12953)
     - 当遇到写冲突时，在报错信息中添加错误码，以方便对冲突原因进行诊断 [#12878](https://github.com/pingcap/tidb/pull/12878)
 + DDL
-    - 为避免误操作，TiDB 默认不再允许删除列的 `AUTO INCREMENT` 属性，当确实需要删除时，请更改系统变量 `tidb_allow_remove_auto_inc`；相关文档请见：[TiDB 专用系统变量和语法](/dev/reference/configuration/tidb-server/tidb-specific-variables.md) [#12146](https://github.com/pingcap/tidb/pull/12146)
+    - 为避免误操作，TiDB 默认不再允许删除列的 `AUTO INCREMENT` 属性，当确实需要删除时，请更改系统变量 `tidb_allow_remove_auto_inc`；相关文档请见：[TiDB 专用系统变量和语法](https://pingcap.com/docs-cn/dev/reference/configuration/tidb-server/tidb-specific-variables/) [#12146](https://github.com/pingcap/tidb/pull/12146)
```

Comment:
@YiniXu9506 This link is broken if we follow our link rules. It's another weird case. PTAL what's the cause. Thanks!